src/sentry/data_export/tasks.py

```diff
@@ -115,10 +115,7 @@ def assemble_download(
                 # module is able to do, it will NOT write bytes like in py2).
                 # Because of this we use the codec getwriter to transform our
                 # file handle to a stream writer that will encode to utf8.
-                if six.PY2:
-                    tfw = tf
-                else:
-                    tfw = codecs.getwriter("utf-8")(tf)
+                tfw = codecs.getwriter("utf-8")(tf)
```

Comment:
We should clean up the comment above too.

Actually I'm pretty sure this whole bit of code becomes a lot simpler if we open the file in text mode as utf-8.

So maybe actually we should leave this for now until we can clean up?

Reply:
I'd still remove the `PY2` check as it is irrelevant. For the rest, I'll leave the comment

Reply:
Yeah, please leave some TODO(py3) type comments :)